### PR TITLE
handle action trace v1

### PR DESCRIPTION
**Pull request by danielAlvess**

### CODE REVIEW

File: examples/connection.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const { Serialize } = require('eosjs');
 const { StaticPool } = require('node-worker-threads-pool');
 const Connection = require('../src/connection');
 const serialize = require('../src/serialize');
-const deserialize = require('../src/deserialize');
+// const { deserialize } = require('../src/deserialize');
```
Comment (porkchop) [inline examples/connection.js:6]: can we remove this if it is unused please?

**Review by porkchop — CHANGES_REQUESTED** [review]
Good idea @danielAlvess . One very minor nitpick in there for you. Also, can you bump the version in packge.json and package-lock.json from `2.1.2` to `2.2.0` as well please? I'll merge and publish the changes when these are done

**Comment by danielAlvess** [comment]
Fixed and bump to version 2.2.0

**Comment by porkchop** [comment]
thanks @danielAlvess LGTM!

**Pull request merged**